gammapy/astro/darkmatter/profiles.py

```diff
@@ -87,6 +88,59 @@ def integrate_spectrum_separation(self, func, xmin, xmax, separation, ndecade):
         return val.sum()
 
 
+class ZhaoProfile(DMProfile):
+    r"""Zaho Profile.
+    .. math::
+        \rho(r) = \rho_s \frac{r_s}{r}^\gamma \left(1 + \frac{r}{r_s}^\alpha \right)^{\frac{\beta - \gamma}{\alpha}}
```

Comment:
Have you check locally that the formula is displayed correctly?

Reply:
Yes it, but I think I should add some parentheses to the fraction of r/rs to make it more readable

Reply:
It compiled but the equation was wrong. Should be correct now